docs/data/pages.ts

```diff
@@ -65,7 +65,17 @@ const pages: MuiPage[] = [
           { pathname: '/x/react-data-grid/row-pinning', plan: 'pro' },
         ],
       },
-      { pathname: '/x/react-data-grid/editing' },
+      {
+        pathname: '/x/react-data-grid/editing-group',
+        title: 'Editing',
+        children: [
+          { pathname: '/x/react-data-grid/editing', title: 'Overview' },
+          { pathname: '/x/react-data-grid/editing/persistence' },
+          { pathname: '/x/react-data-grid/editing/custom-edit-component' },
+          { pathname: '/x/react-data-grid/editing/editing-events' },
+          { pathname: '/x/react-data-grid/editing/recipes' },
```

Comment:
We didn't reach an agreement on "Examples" vs "Recipes" in https://github.com/mui/mui-x/issues/7762 😅

I kept the previous title ("Recipes") for now, feel free to vote on this comment if you have a preference:
- 🚀 for "Recipes"
- 🎉 for "Examples"

Reply:
@richbustos What do you think about this one?